contracts/protocol/tokenization/NTokenApeStaking.sol

```diff
@@ -96,4 +99,47 @@ abstract contract NTokenApeStaking is NToken {
         // should be overridden
         return 0;
     }
+
+    function initializeStakingData() internal {
+        ApeStakingLogic.APEStakingParameter
+            storage dataStorage = apeStakingDataStorage();
+        ApeStakingLogic.executeSetUnstakeApeHFLimit(dataStorage, 1.1e18);
+        ApeStakingLogic.executeSetUnstakeApeIncentive(dataStorage, 100);
+    }
+
+    function setUnstakeApeHFLimit(uint256 hfLimit) external onlyPoolAdmin {
```

Comment:
I think threshold is more understandable

Reply:
+1